+Base/GetClasses.m

```diff
@@ -6,7 +6,6 @@
 if any(arrayfun(@(d)strcmpi(fullfile(d.folder,d.name),target),dir))
     files(1) = []; %assume first entry is the present working directory  and is thus redundant
 end
-assert(length(files)==1,sprintf('Did not find a single match for %s - found %i',target,length(files)))
```

Comment:
Empty file lists were already handled later. This assert breaks things rather than returning info to the user.

Reply:
this was actually intended to prevent further errors with more than 1 file returned

Reply:
Fixed to > 1 in recent commits.